pkg/debug/Dockerfile

```diff
@@ -63,13 +63,54 @@ ADD https://github.com/pixel/hexedit/archive/refs/tags/1.5.tar.gz ../1.5.tar.gz
 RUN tar -C .. -xzvf ../1.5.tar.gz
 RUN ./autogen.sh && ./configure && make DESTDIR=/out install
 
+WORKDIR /usr/src
+ADD https://cdn.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.8p1.tar.gz /usr/src
```

Comment:
sig verify is appreciated.

Reply:
done: https://github.com/lf-edge/eve/pull/4042/files#diff-eb8b8d5cc8d10e1bd74ad66ca8bb1565b32a21a18220ef586c0a22d2e961aee1R75

Reply:
awesome!